scripts/utilities/getPackagePath/index.test.js

```diff
@@ -1,20 +1,50 @@
 const getPackagePath = require('.');
+const getPackages = require('../getPackages');
 
-jest.mock('../getPackages', () => () => [
-  '/psammead/packages/components/psammead-brand',
-  '/psammead/packages/components/psammead-image',
-]);
+jest.mock('../getPackages');
 
 describe('getPackagePath', () => {
-  it('should return the path of the package', () => {
-    const packagePath1 = getPackagePath('psammead-brand');
-    const packagePath2 = getPackagePath('psammead-image');
+  describe('when locally', () => {
+    beforeEach(() => {
+      getPackages.mockImplementation(() => [
+        '/psammead',
+        '/psammead/packages/components/psammead-brand',
+        '/psammead/packages/components/psammead-image',
+      ]);
+    });
 
-    expect(packagePath1).toEqual(
-      '/psammead/packages/components/psammead-brand',
-    );
-    expect(packagePath2).toEqual(
-      '/psammead/packages/components/psammead-image',
-    );
+    it('should return the path of the package', () => {
+      expect(getPackagePath('psammead')).toEqual(
+        '/psammead',
+      );
```

Comment:
prettier is saying this should be single line

Reply:
Done :)